api/player.go

```diff
@@ -44,11 +43,17 @@ func CreatePlayerHandler(app *App) func(c echo.Context) error {
 			return FailWith(http.StatusBadRequest, err.Error(), c)
 		}
 
+		var transaction interfaces.Transaction
+		transaction, err = app.BeginTrans(c.StdContext(), logger)
+		if err != nil {
+			return FailWith(http.StatusBadRequest, err.Error(), c)
```

Comment:
If we can't begin a transaction it is not a client error. It should be an Internal Server Error.